ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/qml/ManageOfflineDataView.qml

```diff
@@ -118,6 +124,19 @@ Page {
             visible: SampleManager.downloadInProgress
             clip: true
         }
+
+        Button {
+            // PortalItem::fetchData does not have a cancel method so we can only clear the remaining items from the download queue
+            text: SampleManager.cancelDownload ? "Cancelling remaining downloads..." : "Cancel remaining downloads"
+            Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
+            visible: SampleManager.downloadInProgress
+            enabled: !SampleManager.cancelDownload
+            onClicked: {
+                console.log("attempting to cancel download");
```

Comment:
you've got a couple of console.logs in here, perhaps want to clean them up unless they are important to keep